crates/wasm-encoder/src/imports.rs

```diff
@@ -40,27 +40,21 @@ impl ImportSection {
     }
 
     /// Define an import.
-    pub fn import(&mut self, module: &str, name: &str, ty: impl Into<ImportType>) -> &mut Self {
+    pub fn import(
+        &mut self,
+        module: &str,
+        name: Option<&str>,
```

Comment:
The explainer says that it is the module name that is optional now, not the field name. Although with the desugaring of two level imports into a single level instance import, it makes more sense to have the field name optional. I'll file an upstream bug for discussion.

Reply:
https://github.com/WebAssembly/module-linking/issues/18

Reply:
Ah yeah I've implemented everything so far as optional fields rather than an optional module, but I'll change things depending on the outcome of that issue.